content/docs/actions.md

```diff
@@ -11,11 +11,20 @@ weight: 9
 You can probably find yourself in a situation where some part of the packit workflow needs to be
 tweaked for your package.
 
-Packit supports actions, a way to change the default implementation for a command
-of your choice.  Packit is able to execute multiple commands. Each action accepts
-a list of commands. By default, the commands are executed directly and
-not in a shell - if you need a shell, just wrap your command like this: `bash -c
-"my fancy $command | grep success"`.
+Packit supports actions, a way to change the default implementation for a
+command of your choice. Packit is able to execute multiple commands for a
+single action. Each action accepts a list of commands. By default, the commands
```

Comment:
To avoid overloading the term *command*.
```suggestion
Packit supports actions, which can be used to change the default implementation of some of the steps in the workflow. Packit is able to execute multiple commands for a
single action. Each action accepts a list of commands. By default, the commands
```

Reply:
neat! I wasn't happy about the initial wording, yours fits well